embulk-input-s3/src/main/java/org/embulk/input/s3/AlwaysRetryable.java

```diff
@@ -18,10 +24,17 @@
 class AlwaysRetryable<T> implements Retryable<T>
```

Comment:
Please change this class name, something like DefaultRetryable. The current name is misleading as it is not "always" anymore.